cli-plugins/apps/tutorials/get-workload-status.hbs.md

```diff
@@ -0,0 +1,635 @@
+      5 + |  labels:
+      6 + |    apps.tanzu.vmware.com/workload-type: web
+      7 + |  name: tanzu-java-web-app
+      8 + |  namespace: default
+      9 + |spec:
+     10 + |  source:
+     11 + |    git:
+     12 + |      ref:
+     13 + |        tag: tap-1.6.0
+     14 + |      url: https://github.com/vmware-tanzu/application-accelerator-samples
+     15 + |    subPath: tanzu-java-web-app
+❓ Do you want to create this workload? [yN]: y
+👍 Created workload "tanzu-java-web-app"
+
+To see logs:   "tanzu apps workload tail tanzu-java-web-app --timestamp --since 1h"
+To get status: "tanzu apps workload get tanzu-java-web-app"
+
+Waiting for workload "tanzu-java-web-app" to become ready...
++ tanzu-java-web-app-build-1-build-pod › prepare
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T11:28:12.746812119-05:00 Build reason(s): CONFIG
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T11:28:12.747149910-05:00 CONFIG:
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T11:28:12.747186215-05:00       + env:
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T11:28:12.747194864-05:00       + - name: BP_OCI_SOURCE
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T11:28:12.747201214-05:00       +   value: tap-1.6.0/bb9e655b33e39b242b6e5d9e218578e75c1d833c
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T11:28:12.747206421-05:00       resources: {}
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T11:28:12.747211761-05:00       - source: {}
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T11:28:12.747217114-05:00       + source:
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T11:28:12.747222613-05:00       +   blob:
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T11:28:12.747228733-05:00       +     url: http://fluxcd-source-controller.flux-system.svc.cluster.local./gitrepository/default/tanzu-java-web-app/bb9e655b33e39b242b6e5d9e218578e75c1d833c.tar.gz
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T11:28:12.747235497-05:00       +   subPath: tanzu-java-web-app
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T11:28:12.747240321-05:00 Loading registry credentials from service account secrets
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T11:28:12.747402066-05:00 Loading secret for "gcr.io" from secret "registry-credentials" at location "/var/build-secrets/registry-credentials"
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T11:28:12.747417620-05:00 Loading secret for "https://gcr.io/" from secret "registry-credentials" at location "/var/build-secrets/registry-credentials"
+...
+```
+
+## Export a workload
+
+### Using --export flag
+
+This flag is used to retrieve the workload definition with all the extraneous,
+cluster-specific, properties and values removed. For example, the status and metadata text boxes like
+`creationTimestamp`. This will permit to apply the workload definition to a different environment 
```

Comment:
`creationTimestamp`. The clean workload definition export is suitable for gitHub commit and/or for application to a different environment.

Reply:
339-343

Reply:
changed